dev-docs/publisher-api-reference.md

```diff
@@ -14,7 +14,7 @@ pid: 10
 This page has documentation for the public API methods of Prebid.js.
 
 {: .alert.alert-danger :}
-Methods marked as deprecated will be removed in version 1.0 (scheduled for release Q4 2017).  
+Methods marked as deprecated will be removed in version 1.0 (scheduled for release Q4 2017).
```

Comment:
We can say "Methods marked as deprecated were removed in version 1.0" and can link to the release docs.

Reply:
Cool, updated in 83e1042